### PR TITLE
Clarify grammar

### PR DESCRIPTION
**Pull request by jsatk**
Some suggested phrasing & grammar updates.

### CODE REVIEW

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md [hunk 1]
```diff
@@ -110,7 +113,7 @@ The Tokens not available in the release won't be used/shown.
 ### Series Folder Format
 
 ```bash
-{Series TitleYear}
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:113]: This exists.

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md [hunk 2]
```diff
@@ -76,7 +79,7 @@ The Tokens not available in the release won't be used/shown.
 > **All the details**
 
 ```bash
-{Series TitleYear} - S{season:00}E{episode:00} - {absolute:000} - {Episode CleanTitle} [{Preferred Words }{Quality Full}]{[MediaInfo VideoDynamicRange]}[{MediaInfo VideoBitDepth}bit]{[MediaInfo VideoCodec]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}{-Release Group}
+{Series Title} - S{season:00}E{episode:00} - {absolute:000} - {Episode CleanTitle} [{Preferred Words }{Quality Full}]{[MediaInfo VideoDynamicRange]}[{MediaInfo VideoBitDepth}bit]{[MediaInfo VideoCodec]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}{-Release Group}
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:82]: exists

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md [hunk 3]
```diff
@@ -52,7 +55,7 @@ The Tokens not available in the release won't be used/shown.
 > **All the details**
 
 ```bash
-{Series TitleYear} - {Air-Date} - {Episode CleanTitle} [{Preferred Words }{Quality Full}]{[MediaInfo VideoDynamicRange]}[{MediaInfo VideoBitDepth}bit]{[MediaInfo VideoCodec]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}{-Release Group}
+{Series Title} - {Air-Date} - {Episode CleanTitle} [{Preferred Words }{Quality Full}]{[MediaInfo VideoDynamicRange]}[{MediaInfo VideoBitDepth}bit]{[MediaInfo VideoCodec]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}{-Release Group}
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:58]: Ditto

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md [hunk 4]
```diff
@@ -16,7 +19,7 @@ The Tokens not available in the release won't be used/shown.
 > **All the details**
 
 ```bash
-{Series TitleYear} - S{season:00}E{episode:00} - {Episode CleanTitle} [{Preferred Words }{Quality Full}]{[MediaInfo VideoDynamicRange]}[{MediaInfo VideoBitDepth}bit]{[MediaInfo VideoCodec]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}{-Release Group}
+{Series Title} - S{season:00}E{episode:00} - {Episode CleanTitle} [{Preferred Words }{Quality Full}]{[MediaInfo VideoDynamicRange]}[{MediaInfo VideoBitDepth}bit]{[MediaInfo VideoCodec]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}{-Release Group}
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:22]: Ditto

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
-first of it's personal preference, but the support staff often recommend to add also non recoverable info.
+Often on the [Sonarr Discord][sd] people ask what's the recommended/best way to
+name your files.  First off it's personal preference, but the support staff
+often recommend to add also non- recoverable info.
 
 Why?
 
-if you even need to do a re-install, re-import for what ever reason in Radarr or Plex/Emby/JellyFin it's nice to have all that info in the filename so it get's imported correctly and not been matched as HDTV or WEB-DL etc.
+If, for what ever reason, you ever need to do a re-install or re-import in
+Radarr or Plex/Emby/JellyFin it's nice to have all that info in the filename so
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:10]: Sonarr*

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md [hunk 6]
```diff
@@ -1,11 +1,14 @@
 # Recommended naming scheme
 
-Often on Sonarr discord people asking what's the recommended/best way to naming your files.
-first of it's personal preference, but the support staff often recommend to add also non recoverable info.
+Often on the [Sonarr Discord][sd] people ask what's the recommended/best way to
+name your files.  First off it's personal preference, but the support staff
+often recommend to add also non- recoverable info.
```
Comment (bakerboy448) [inline docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:5]: *it’s often recommended to add non-recoverable

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
Also update your commit message then to just “grammar & style fixes”

**Comment by jsatk** [comment]
My bad @bakerboy448. Will remove the {TitleYear} changes.

File: docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md (hunk 5 above)
Comment (jsatk) [reply docs/Sonarr/V3/Sonarr-recommended-naming-scheme.md:10]: Good catch!

**Pull request merged**